internal/agent-api/client.go

```diff
@@ -27,7 +27,7 @@ type ContactLostCallback func(string)
 
 const (
 	defaultAgentPingIntervalMillis = 5000
-	maxRetryAttempts               = 3
+	maxRetryAttempts               = 10
```

Comment:
does this actually need to be 10?? This seems pretty excessive and it we're waiting 10 retries for the agent, then something bigger is happening than retries.

Reply:
Yes... 3 wasn't enough. At 100ms intervals, it's only 1 second.

Reply:
I'm not sure I agree. We want the request to succeed as soon as possible. I don't see a downside in keeping the attempt count high and the timeout at 100ms at this time.